framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2218,6 +2218,30 @@ protected boolean testTypevarContainment(
     }
 
     /**
+     * Create an OverrideChecker.
+     *
+     * <p>This exists so that subclasses can subclass OverrideChecker and use their subclass instead
+     * of using OverrideChecker itself.
+     */
+    protected OverrideChecker makeOverrideChecker(
+            Tree overriderTree,
+            AnnotatedExecutableType overrider,
+            AnnotatedTypeMirror overridingType,
+            AnnotatedTypeMirror overridingReturnType,
+            AnnotatedExecutableType overridden,
+            AnnotatedDeclaredType overriddenType,
+            AnnotatedTypeMirror overriddenReturnType) {
+        return new OverrideChecker(
+                overriderTree,
+                overrider,
+                overridingType,
+                overridingReturnType,
+                overridden,
+                overriddenType,
+                overriddenReturnType);
+    }
+
+    /**
      * Type checks that a method may override another method. Uses the OverrideChecker class.
```

Comment:
Please adapt this javadoc to point to createOverrideChecker instead of the standard OverrideChecker class.